pages/services/kafka-zookeeper/2.3.0-3.4.12/release-notes/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Release Notes
+menuWeight: 120
+model: /services/kafka-zookeeper/data.yml
+render: mustache
+---
+
+# Version 2.3.0-3.4.12
+
+## New Features
```

Comment:
`s/New //` here?

Reply:
`New Features` is used in some of the other services, but I went for local consistency. I also corrected an incorrect version replace in f3c23df

Reply:
👍